### PR TITLE
Allow for initial CCS config to be created for DID-Registry

### PR DESCRIPTION
**Pull request by dwendland**

### CODE REVIEW

**Review by pulledtim — APPROVED** [review]

File: charts/business-api-ecosystem/values.yaml [hunk 1]
```diff
@@ -58,15 +58,26 @@ siop:
...
+    scopes:
+      defaultScope:
+        - type: "ParticipantRegistryDefaultCredential"
+          trustedParticipantsLists: [
+            "https://tir.fiware.dev"
+          ]
+          trustedIssuersLists: [
+            "https://til.fiware.dev"
+          ]
+      didRead:
```
Comment (pulledtim) [inline charts/business-api-ecosystem/values.yaml:73]: Do we need this one? TIR DIDRegistry is not used and the TIR API doesn't care for this scope

File: charts/trusted-issuers-registry/templates/configmap.yaml [hunk 2]
```diff
@@ -30,4 +30,26 @@ data:
...
+  init.sh: |-
+    curl -X 'POST' \
+      '{{ .Values.ccs.endpoint }}service' \
+      -H 'accept: */*' \
+      -H 'Content-Type: application/json' \
+      -d '{
+      "id": "{{ .Values.ccs.id }}",
+      "default_scope": "{{ .Values.ccs.defaultScope }}",
+      {{- if .Values.ccs.scopes -}}
+      "scopes": {{- toJson .Values.ccs.scopes }}
```
Comment (pulledtim) [inline charts/trusted-issuers-registry/templates/configmap.yaml:52]: Is the name of the field not oidcScopes? https://github.com/FIWARE/credentials-config-service/blob/main/api/credentials-config-service.yaml#L234

File: charts/trusted-issuers-registry/templates/configmap.yaml [hunk 3]
```diff
@@ -30,4 +30,26 @@ data:
...
+    {{- include "tir.labels" . | nindent 4 }}
+data:
+  init.sh: |-
+    curl -X 'POST' \
+      '{{ .Values.ccs.endpoint }}service' \
+      -H 'accept: */*' \
+      -H 'Content-Type: application/json' \
+      -d '{
+      "id": "{{ .Values.ccs.id }}",
+      "default_scope": "{{ .Values.ccs.defaultScope }}",
```
Comment (pulledtim) [inline charts/trusted-issuers-registry/templates/configmap.yaml:50]: defaultOidcScope

File: charts/business-api-ecosystem/templates/biz-ecosystem-logic-proxy/configmap.yaml [hunk 4]
```diff
@@ -13,6 +13,9 @@ data:
       -H 'Content-Type: application/json' \
       -d '{
       "id": "{{ .Values.siop.clientId }}",
-      "credentials": {{- toJson .Values.siop.ccs.credentials }}
+      "default_scope": "{{ .Values.siop.ccs.defaultScope }}",
```
Comment (pulledtim) [inline charts/business-api-ecosystem/templates/biz-ecosystem-logic-proxy/configmap.yaml:16]: same here

File: charts/business-api-ecosystem/values.yaml (hunk 1 above)
Comment (dwendland) [reply charts/business-api-ecosystem/values.yaml:73]: It's just meant to have an example on how to fill this scopes object. In the end it's up to the final config what to put there.

File: charts/trusted-issuers-registry/templates/configmap.yaml (hunk 2 above)
Comment (dwendland) [reply charts/trusted-issuers-registry/templates/configmap.yaml]: You are right, this was not up-to-date in the ticket.
Fixed!

**Pull request merged**